ncm-openstack/src/main/pan/components/openstack/schema.pan

```diff
@@ -0,0 +1,28 @@
+${componentschema}
+
+include 'quattor/schema';
+include 'pan/types';
+include 'components/openstack/common';
+include 'components/openstack/keystone';
+include 'components/openstack/nova';
+include 'components/openstack/glance';
+include 'components/openstack/neutron';
+include 'components/openstack/horizon';
+
+@documentation {
+Type to define OpenStack services
+Keystone, Nova, Neutron, etc
+}
+type component_openstack = {
+    include structure_component
+    'keystone' : openstack_keystone_config
```

Comment:
i don't like this structure. we should use a substructure with the types, eg
```pan
type openstack_identity_config = {
    keystone ....
}

type component_openstack = {
    ...
    `identity' : openstack_identity_config
    ...
}
```

move all things nova under compute, neutron under network ....